packages/chain-adapters/src/types.ts

```diff
@@ -83,43 +83,16 @@ export type SubscribeTxsInput = {
   accountType?: UtxoAccountType
 }
 
-export type TxFee = {
-  assetId: string
-  value: string
-}
+export type TransferType = unchained.TransferType
+export type TradeType = unchained.TradeType
 
-export type Transaction<T extends ChainId> = {
-  address: string
-  blockHash?: string
-  blockHeight: number
-  blockTime: number
+export type Transaction<T extends ChainId> = Omit<unchained.StandardTx, 'transfers'> & {
```

Comment:
Will remove the generic as the `chain` field we use it for is unused

Reply:
https://github.com/shapeshift/lib/pull/944/commits/820d2084349f986dbd9d0e0c1086e004ec68e572